src/python/artm_vs2012.sln

```diff
@@ -1,62 +0,0 @@
-﻿
```

Comment:
I know @MelLain  is using Visual Studio as a tool to develop for python. Please, ask him if this is OK to remove the files. To me this is OK - I have switched to PyCharm, and I don't need any project files on disk.